www/include/views/componentTemplates/formComponentTemplate.ihtml

```diff
@@ -34,6 +34,9 @@
 			{if $o == "a" || $o == "c"}
 				<td id="td_list_metrics" style="padding: 0px; border-width: 0px;">
 					&nbsp;<img src="./img/icons/arrow-left.png" class="ico-14" onclick="insertValueQuery()" />&nbsp;
+				    <td class="log-select" style="padding-bottom: 4px">
+					    <select id="sl_list_metrics" size="1"></select>
+				    </td>
```

Comment:
You should not include a column (td) in an existing column.

Reply:
@callapa done :)